bin/clang/astVisitor.cc

```diff
@@ -2647,13 +2630,15 @@ SkeletonASTVisitor::VisitTypedefDecl(TypedefDecl* D)
     if (!str_ty){
       errorAbort(D, *ci_,
                  "structure type did not return a record declaration");
+      return false;
```

Comment:
change to:
````
if (str_ty){
  ...
}  else {
 errorAbort
}
````
remove the false return

Reply:
I don't actually like this, I think the following is fine:

```c++
if(!str_ty){
  errorAbort()
}
// use str_ty
```
Specifically because it has as little nesting as possible.  Nesting the use inside of an else seems unnecessary.  This is also how llvm does their checks.

Reply:
That's fine.